### DISCUSSION TITLE
Other possibility to connect?

### DISCUSSION THREAD

**Discussion by SchneHa** [bikemike/IntexSpaManager#4] [Ideas]
Hi Mike,
thank you for your description and the clear pictures. 
As I can see, your mainboard is covered with silicone. Same as mine. My PureSpa is pretty new and I try to avoid loosing the warranty. Therefore I'm looking for a possibility to connect the D1 mini without cutting the silicone cover.
Nearby is a connector of 5 pins (near "Test" pins) labeled "- PA0 PA4 RET +". It seems to be the programming interface of the internal MC of the Mainboard. Do you think that this could be a possibility to connect the D1?
Best regards,
Hans

**Comment by bikemike** [comment]
It may be possible that they have some sort of control and monitoring protocol available on those pins but I'm not sure how you'd figure that out.

If you don't want to void your warranty you could could do something like this user did:

https://github.com/YorffoeG/diyscip
